broadinstitute/pearl
# JN-319: adding footer to ourhealth surveys

Nick, I'd love your thoughts on the best way to get this to play nicely with the surveyJS styling.  I've hacked it in, with some negative margin to offset the 80px margin that SurveyJS includes at the bottom of its forms.  Also, I wasn't sure the best way to match surveyJS widths.  As best as I can tell, the width of the survey comes from `max-width: calc(84 * var(--base-unit, 8px));` set on the `.sd-body.sd-body-static` elements.  I've just approximated it as a maxWidth of 600, not sure if that's close enough.

![image](https://user-images.githubusercontent.com/2800795/234628726-9f0a85ca-2c38-45b3-a03a-93e29a2e93a4.png)

(now with improved styling from Erin)

TO TEST:
1. restart adminApiApp
2. run `./scripts/populate_portal.sh ourhealth`
3. sign into participant view as jsalk
4. click on any survey (except medications)
5. observe a footer on the last page
